addons/mail/static/src/core/common/sound_effects_service.js

```diff
@@ -22,7 +22,9 @@ export class SoundEffects {
             undeafen: { defaultVolume: 0.15, path: "/mail/static/src/audio/undeafen_new_01" },
             unmute: { defaultVolume: 0.2, path: "/mail/static/src/audio/unmute_1" },
         };
+        this.setup();
     }
+    setup() {} // Empty setup which can be overridden when it's needed
```

Comment:
Again, you can just move everything in `setup()` function, and make constructor just call `setup()`. That way, setup is always used so it doesn't feel off to have empty function just for patching purposes